packages/react-pdf-selection/src/components/PdfViewer.tsx

```diff
@@ -161,7 +161,7 @@ export class PdfViewer<D extends object> extends Component<PdfViewerProps<D>, Pd
         for (const page of pages) {
             const width = page.view[2];
             const height = page.view[3];
-            const orientation = page.rotate === 90 ? PDFOrientation.LANDSCAPE : PDFOrientation.PORTRAIT;
+            const orientation = page.rotate === 270 ? PDFOrientation.LANDSCAPE : PDFOrientation.PORTRAIT;
```

Comment:
I think it should be

```suggestion
            const orientation = Math.abs(page.rotate) === 90 || Math.abs(page.rotate) === 270 ? PDFOrientation.LANDSCAPE : PDFOrientation.PORTRAIT;
```
since according to [metadata2go](https://www.metadata2go.com/file-info/page-rotation) 90 and -90 are possible valid values for landscape